contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -773,23 +774,20 @@ contract LiquidityProtection is ILiquidityProtection, Utils, Owned, ReentrancyGu
      *
      * @param id position id
      * @param newProvider the new provider
-     * @param target the contract to notify
-     * @param data the data to call the contract with
+     * @param callback the callback contract to notify
+     * @param data custom data provided to the callback
      *
      * @return new position id
      */
-    function transferPositionAndCall(
+    function transferPositionAndNotify(
         uint256 id,
         address newProvider,
-        address target,
-        bytes memory data
-    ) external nonReentrant validAddress(newProvider) validAddress(target) returns (uint256) {
-        // make sure that we're not trying to call into the zero address or a fallback function
-        require(data.length >= FUNC_SELECTOR_LENGTH, "ERR_INVALID_CALL_DATA");
```

Comment:
Should also remove the declaration of the `FUNC_SELECTOR_LENGTH` constant, wherever it is.

Reply:
Oh, right! It's at the beginning of the file, with the rest of the constants.